### DISCUSSION TITLE
Can't export model with Cast operator to NNEF?

### DISCUSSION THREAD

**Discussion by davidatsurge** [sonos/tract#695] [Q&A] [answered]
``` 
$ /Users/davidat/git/tract/target/debug/tract ../EfficientConformer/model.onnx dump --nnef optimized.nnef.tar.gz
```

gives me

```
[2022-04-26T17:43:31.508911000Z ERROR tract] No serializer found for node #89 "Cast_842" Cast
```

Is this a known limitation?

 The reason I want to export to NNEF is because I would like to see if the `.into_optimized()` step can be done much quicker (it takes >1min right now) if I load the model from NNEF format instead of ONNX. Might be worth noting also that the step is much much quicker when I do a `--release` build.

**Comment by kali** [comment, marked as answer]
Try to add "--nnef-tract-core" (before dump) to enable tract nnef extensions. If that does not work, I'll need the model to have a look :)

**Reply by davidatsurge** [reply]
That seems to have taken care of it!

**Reply by kali** [reply]
We need an extension to nnef here as type semantics from nnef are sloppier than what they are in tract/onnx/tf...
